aikau/src/main/resources/alfresco/upload/AlfUploadDisplay.js

```diff
@@ -216,12 +213,12 @@ define(["dojo/_base/declare",
          var failedFile = domConstruct.create("tr", {
             className: "file"
          }, this.failedItemsNode);
-         var fileName = domConstruct.create("td", {
+         domConstruct.create("td", {
              innerHTML: fileName,
              className: "filename"
          }, failedFile);
-         var reason = domConstruct.create("td", {
-             innerHTML: (error != null && error.reason != null) ? error.reason : message.get("unknown.failure.reason"),
+         domConstruct.create("td", {
+             innerHTML: (error && error.reason) ? error.reason : this.message("unknown.failure.reason"),
```

Comment:
Could we need to `this.message` the `error.reason`?? As usual, presume this is correct, but just double-checking.